### PR TITLE
[onednn] Bumped version to 3.0.0 release.

### PR DESCRIPTION
**Pull request by as-shura**
@microsoft-github-policy-service agree

### CODE REVIEW

File: ports/onednn/vcpkg.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "onednn",
-  "version-semver": "2.6.1",
+  "version-semver": "3.0.0",
```
Comment (LilyWangLL) [inline ports/onednn/vcpkg.json:3]:
```suggestion
  "version": "3.0.0",
```
Is this port a semver library? If it is not, here should use `"version"`?

File: ports/onednn/portfile.cmake [hunk 2]
```diff
@@ -1,8 +1,8 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO oneapi-src/oneDNN
-    REF 58be3660fb57c4c4a3d306730e849237d1271572 #2.6.1
-    SHA512 8fc51655d0ecdb5b137d04c6b3ee01959d9505e0d7ee74cab872447b5873f214d38db55edf4040a56d4455557ec2c0707097337c6e276dbaf1c9c6e48b95a820
+    REF 3fef24ec08eb89a2c4be4425162c0651af7f1de3 #3.0
```
Comment (LilyWangLL) [inline ports/onednn/portfile.cmake:4]:
```suggestion
    REF "v${VERSION}"
```
Here suggest use `"v${VERSION}"`, it could avoid change in next update.

**Review by LilyWangLL — CHANGES_REQUESTED** [review]

**Comment by LilyWangLL** [comment]
Please also remove `libcopp:arm-uwp=fail` from [VCPKG_ROOT]/scripts/ci.baseline.txt, there is `"supports": "x64 & !uwp"` in `vcpkg.json`.

**Comment by LilyWangLL** [comment]
> _This is a new experimental fast check for PR issues. Please let us know if this bot is helpful!_
> 
> After committing all other changes, the version database must be updated
> ```shell
> git add -u && git commit
> git checkout 91dd61bd441a68b4017b61011d0350b2e6aeeccf -- versions
> ./vcpkg x-add-version --all
> ```
> 

Please update the version database.

**Comment by as-shura** [comment]
> Please also remove `libcopp:arm-uwp=fail` from [VCPKG_ROOT]/scripts/ci.baseline.txt, there is `"supports": "x64 & !uwp"` in `vcpkg.json`.

Done! Is there anything else ?

**Comment by as-shura** [comment]
> ```shell
> git add -u && git commit
> git checkout 91dd61bd441a68b4017b61011d0350b2e6aeeccf -- versions
> ./vcpkg x-add-version --all
> ```

I just did that now check if ok please.

**Review by LilyWangLL — DISMISSED** [review]

**Review by BillyONeal — APPROVED** [review]
Thanks for the update!

**Comment by BillyONeal** [comment]
> Please also remove `libcopp:arm-uwp=fail` from [VCPKG_ROOT]/scripts/ci.baseline.txt, there is `"supports": "x64 & !uwp"` in `vcpkg.json`.

I think some wires got crossed here :D

https://dev.azure.com/vcpkg/public/_build/results?buildId=84590

REGRESSION: libcopp:arm-uwp failed with BUILD_FAILED. If expected, add libcopp:arm-uwp=fail to C:\a\2\s\scripts\azure-pipelines/../ci.baseline.txt.

I'm fixing

**Pull request merged**